### ISSUE TITLE
For tag matching using non space delimiter so can match strings containing spaces

### ISSUE THREAD

**Issue by zedster** [danielquinn/paperless#55]
eg: "credit card"   
if I search for credit and card seperately i get lots of matches that aren't relavent.
Similar if I search for a company name with common words eg: "Direct Line".

I had a look at regex expressions to match a string with spaces for example the two above, in order.I couldn't find any obvious examples. And it'd seem a bit overkill. if you use a comma , or pipe | might be better seperators.

Also related, I could only see the help text and the fact space delimitation was used by looking at the code.

**Comment by danielquinn** [comment]
Currently, the search feature is the default used by Django's admin system, so I don't have a lot of power over how that is executed.  I'm working on the API right now though, and when that's ready, it'll allow searches using syntax most people are used to, ie. `"joining words with quotes" while other words are separate`.  It'll be an inclusive match though, since I don't have the skill to write any kind of fuzzy matching logic.

In other words, to find documents with `alpha` and `beta` next to each other, you'd search for `"alpha beta"`, and if you just want documents with those words in the same document, you'd use `alpha beta`.  At least, that's the current plan.  I'll have everything documented once the API is finished.
